components/mpas-ocean/cime_config/buildnml

```diff
@@ -283,6 +283,19 @@ def buildnml(case, caseroot, compname):
         if ocn_ismf == 'data':
             data_ismf_file = 'prescribed_ismf_adusumilli2020.ECwISC30to60E2r1.230429.nc'
 
+    elif ocn_grid == 'ECwISC30to60E3r1':
+        decomp_date = '20230815'
+        decomp_prefix = 'mpas-o.graph.info.'
+        restoring_file = 'sss.PHC2_monthlyClimatology.ECwISC30to60E3r1_nomask.XXXXXX.nc'
```

Comment:
We would need Mat to make this for us or better yet reveal his secret magical powers.